.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - 3.6
+  - 3.5
+  - 3.4.2
+install:
+  # Install latest versions of scipy and matplotlib with support for python 3.4
+  # Required for multidiff library
```

Comment:
What are these libraries used for? I don't reacall the code using any of this...

Reply:
While running travis I got errors when these libraries were not installed.
If I comment out this line and run travis I get the following error:

0.79s$ pip install multidiff
Collecting multidiff
  Downloading https://files.pythonhosted.org/packages/16/dd/71ac8edf517da90ca4b5ec455f9e5b9b722365ad3b0aa6a500c6942601f6/multidiff-0.1.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-b6wphd6c/multidiff/setup.py", line 6, in <module>
        import multidiff
      File "/tmp/pip-build-b6wphd6c/multidiff/multidiff/__init__.py", line 1, in <module>
        from .diffusion_matrix import (compute_diffusion_matrix, preprocessing,
      File "/tmp/pip-build-b6wphd6c/multidiff/multidiff/diffusion_matrix.py", line 2, in <module>
        import matplotlib.pyplot as plt
    ImportError: No module named 'matplotlib'

Reply:
While running travis I got errors when these libraries were not installed.
If I comment out this line and run travis I get the following error:

    $ pip install multidiff
    Collecting multidiff
      Downloading 
    https://files.pythonhosted.org/packages/16/dd/71ac8edf517da90ca4b5ec455f9e5b9b722365ad3b0aa6a500c6942601f6/multidiff-0.1.1.tar.gz
        Complete output from command python setup.py egg_info:
        Traceback (most recent call last):
          File "<string>", line 1, in <module>
          File "/tmp/pip-build-b6wphd6c/multidiff/setup.py", line 6, in <module>
            import multidiff
          File "/tmp/pip-build-b6wphd6c/multidiff/multidiff/__init__.py", line 1, in <module>
            from .diffusion_matrix import (compute_diffusion_matrix, preprocessing,
          File "/tmp/pip-build-b6wphd6c/multidiff/multidiff/diffusion_matrix.py", line 2, in 
    <module>
        import matplotlib.pyplot as plt
    ImportError: No module named 'matplotlib'

Reply:
Oh, the multidiff on pip is actually another library altogether. This is not my project but a project by a chemist doing some research. The multidiff in this repository is not in pip or pypi.
 and has to be installed manually.

Not mine, but some chemist researchers: https://pypi.org/project/multidiff/

Reply:
Please review the changes